src/postgres_utils.py

```diff
@@ -0,0 +1,167 @@
+import os
+import subprocess
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import Session
+from psycopg2.extensions import AsIs
+import pandas as pd
+from src.constants import SQL_FILE_DIRECTORY
+
+
+load_dotenv()
+BUILD_ENGINE = os.getenv("SQL_ENGINE_EDM_DATA", "")
```

Comment:
I wasn't able to get this to run without removing the `""` - were you able to view the source data tables as the code is now?

Reply:
hmm yea I was able to. did you stop and start the app after adding the value to the `.env` file? I'm down to remove the `""` placeholder value since maybe we could say no placeholder could ever be valid or useful for this

Reply:
I did not, it works after I restarted the app. I guess I'm lacking an understanding why we would need a placeholder here as supposed to the other instances of when we call `os.getenv`?

Reply:
nah I don't think there's a good reason to have a placeholder here. since there's only one valid `BUILD_ENGINE` aka QAQC database I'll drop the placeholder